serialization/Schemas/spdx-v3.json

```diff
@@ -0,0 +1,1061 @@
+        },
+        "prefixes": {
+          "$ref": "#/definitions/NamespaceMap",
+          "description": "Namespace abbreviations used in this file (rdf PREFIX)"
+        },
+        "created": {
+          "$ref": "#/definitions/DateTime",
+          "description": "File creation info and element defaults"
+        },
+        "creator": {
+          "type": "array",
+          "description": "File creation info and element defaults - Link(Identity)",
+          "minItems": 1,
+          "items": {
+            "$ref": "#/definitions/Element",
+            "description": "File creation info and element defaults - Link(Identity)"
+          }
+        },
+        "specVersion": {
+          "$ref": "#/definitions/SemVer",
+          "description": "File creation info and element defaults"
+        },
+        "profile": {
+          "type": "array",
+          "description": "File creation info and element defaults",
+          "minItems": 1,
+          "items": {
+            "$ref": "#/definitions/ProfileIdentifier",
+            "description": "File creation info and element defaults"
+          }
+        },
+        "dataLicense": {
+          "$ref": "#/definitions/LicenseId",
+          "description": "File creation info and element defaults"
+        },
+        "element": {
+          "type": "array",
+          "description": "Element values, defined in this file or copied",
+          "uniqueItems": true,
+          "minItems": 1,
+          "items": {
+            "$ref": "#/definitions/Element",
+            "description": "Element values, defined in this file or copied"
+          }
+        },
+        "spdxDocumentId": {
+          "$ref": "#/definitions/Element",
+          "description": "Optional SpdxDocument element that describes this file"
+        }
+      }
+    },
+    "Elements": {
+      "title": "Elements",
+      "type": "array",
+      "description": "Serialized set of independent Element values",
+      "uniqueItems": true,
+      "minItems": 1,
+      "maxItems": 100,
+      "items": {
+        "$ref": "#/definitions/Element"
+      }
+    },
+    "NamespaceMap": {
+      "title": "NamespaceMap",
+      "type": "object",
+      "additionalProperties": false,
+      "minProperties": 1,
+      "maxProperties": 100
+    },
+    "Element": {
+      "title": "Element",
+      "type": "object",
+      "additionalProperties": false,
+      "required": [
+        "id",
+        "type"
+      ],
+      "maxProperties": 100,
+      "properties": {
+        "id": {
+          "$ref": "#/definitions/IRI",
+          "description": "Element unique identifier: local, prefix:local, or absolute-IRI"
+        },
+        "type": {
+          "$ref": "#/definitions/ElementType",
+          "description": "Element type and type-specific properties"
+        },
+        "created": {
+          "$ref": "#/definitions/DateTime",
+          "description": "required: explicit or default"
+        },
+        "creator": {
```

Comment:
this is called `createdBy` in the model

Reply:
I was considering proposing a name change to the model, but will fix the schema to agree with the current model.